### PR TITLE
Run codespell to cleanup typos

### PR DESCRIPTION
**Pull request by rhatdan**
[NO NEW TESTS NEEDED]

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
LGTM. If for any reason you need to resubmit (not worth it just for this), could you also clean up my bonehead typo here? https://github.com/containers/podman/blob/7934b77dd5372c22063686a218b8b48c2fcaca8c/hack/xref-helpmsgs-manpages#L301

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

**Comment by mheon** [comment]
Needs a rebase

File: contrib/helloimage/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Using this image is helpful to:
 
  * Prove that basic Podman operations are working on the host.
  * Shows that the image was pulled from the quay.io container registry.
- * Container creation was successfuly accomplished. (`podman ps -a`)
+ * Container creation was successfully accomplished. (`podman ps -a`)
```
Comment (TomSweeneyRedHat) [inline contrib/helloimage/README.md:15]: gah, one of my "I can never type/spell this right" words bites me.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**